examples/nunjucks/components/01-atoms/button/button--link.njk

```diff
@@ -0,0 +1,5 @@
+{% extends '@base-button' %}
+
+{% block class %}text-pink-600 hover:text-pink-500 focus:text-pink-500 font-bold uppercase tracking-wide transition-all duration-200 ease-out group{% if class %} {{ class }}{% endif %}{% endblock class %}
+
+{% block after %}{% render '@icon--tiny', { id: 'chevron-right', class: 'ml-2 transition-all duration-200 ease-out transform group-hover:translate-x-1' }, true %}{% endblock after %}
```

Comment:
i guess a personal preference, but i really really dislike when so much code is on a single line. it's partially inevitable if using tailwind but we could at least put object properties on a newline?

Reply:
Agree, I forgot I could use the `{%- -%}` syntax to keep the template cleaner while removing white-spaces, should be better now.